ut/alarm_req_listener_test.cpp

```diff
@@ -258,8 +267,14 @@ TEST_F(AlarmReqListenerTest, SetAlarm)
   _ms.trap_complete(1, 5);
 }
 
+// We have to advance time here so that our alarms are not filtered out by the
+// alarm_filtered function with the alarm trap sender. This filters out any
+// alarms raised in a repeated state during five seconds of each other (the
+// value of ALARM_FILTER_TIME), even if the state of the alarm changes with that
+// five seconds.
```

Comment:
The key point is that we are using the same `AlarmTrapSender` for all the tests, which means this test doesn't start from a clean state, so we have to wait for the filters to expire before doing our test.